tests/jobs/test_scheduler_job.py

```diff
@@ -3554,31 +3498,6 @@ def run_with_error(ti, ignore_ti_state=False):
         assert ti.try_number == 1
         assert ti.state == State.SUCCESS
 
-    def test_retry_handling_job(self):
-        """
-        Integration test of the scheduler not accidentally resetting
-        the try_numbers for a task
-        """
-        dagbag = DagBag(TEST_DAG_FOLDER, include_examples=False)
-        dagbag.sync_to_db()
-        dag = dagbag.get_dag("test_retry_handling_job")
-        dag_task1 = dag.get_task("test_retry_handling_op")
-        dag.clear()
-        dag.sync_to_db()
-
-        scheduler_job = Job(job_type=SchedulerJobRunner.job_type, heartrate=0)
-        self.job_runner = SchedulerJobRunner(job=scheduler_job, num_runs=1)
-        self.job_runner.processor_agent = mock.MagicMock()
-        run_job(scheduler_job, execute_callable=self.job_runner._execute)
-
-        session = settings.Session()
-        ti = (
-            session.query(TaskInstance)
-            .filter(TaskInstance.dag_id == dag.dag_id, TaskInstance.task_id == dag_task1.task_id)
-            .first()
-        )
-        assert ti.state == State.UP_FOR_RETRY
```

Comment:
This test adds nothing of note anymore, and isn't testing anything about SchedulerJob behaviour